COMPATIBILITY.md

```diff
@@ -54,6 +54,7 @@ In case you find any error, please [create a new issue](https://github.com/packi
 | ----------------------------- | :----: | :----: | :---------------------: |
 | `change_token`                |   ✘    |   ✔   |           ✔            |
 | `get_release`                 |   ✔   |   ✔   |            ✘            |
+| `get_commits`                 |   ✔   |   ✔   |            ✘            |
```

Comment:
Is there something that works on Pagure? (not even push counts, cause even that can yield 500…)